pgo/cmd/update.go

```diff
@@ -56,6 +60,10 @@ func init() {
 	UpdateClusterCmd.Flags().BoolVar(&DisableAutofailFlag, "disable-autofail", false, "Disables autofail capabitilies in the cluster.")
 	UpdateClusterCmd.Flags().BoolVar(&EnableAutofailFlag, "enable-autofail", false, "Enables autofail capabitilies in the cluster.")
 	UpdateClusterCmd.Flags().StringVarP(&Selector, "selector", "s", "", "The selector to use for cluster filtering.")
+	UpdateClusterCmd.Flags().BoolVarP(&DisableStandby, "disable-standby", "", false,
+		"Disables standby mode if enabled in the cluster(s) specified.")
+	UpdateClusterCmd.Flags().BoolVarP(&EnableStandby, "enable-standby", "", false,
+		"Enables standby mode in the cluster(s) specified.")
```

Comment:
Since we recently went the direction of `enable` and `disable` flags for autofail (see https://github.com/CrunchyData/postgres-operator/pull/1245), I wanted to remain consistent here for standby.

I'm not against adopting another approach, but just want to be sure we do it holistically/consistently.